energy_py/envs/battery/battery_env.py

```diff
@@ -142,8 +143,7 @@ def _step(self, action):
         net_charge = action / 12
 
         #  we first check to make sure this charge is within our capacity limit
-        unbounded_new_charge = old_charge + net_charge
-        bounded_new_charge = max(min(unbounded_new_charge, self.capacity), 0)
+        bounded_new_charge = np.clip(old_charge + net_charge, 0, self.capacity)
```

Comment:
That makes sense to me, perhaps also change `bounded_new_charge` to `new_charge` ?